File: iSort/sort.rb
nums = Array.new(100000){rand{0..1000000}}


len = nums.length() - 1
for i in 0..len
	min = i
	for j in i..len
		if nums.nil?
			print nil
		end
		if nums[j] < nums[min]
			min = j
		end
	end
	temp = nums[i]
	nums[i] = nums[min]
	nums[min] = temp
end

File: iSort/sort.c
#include <time.h>
#include <stdlib.h>
#include <stdio.h>
int main(){
	int length = 100000;
	double numbers[length];
	int i, j;

	srand(time(NULL));
	for(i=0;i<length;i++){
		numbers[i] = rand();
	}

	
	for(i=0; i<length; i++){
		int min = i;
		for(j=i;j<length;j++){
			if(numbers[j] < numbers[min]){
				min = j;
			}
		}
		double temp = numbers[i];
		numbers[i] = numbers[min];
		numbers[min] = temp;
	}
}